src/main/resources/com/sonyericsson/hudson/plugins/gerrit/trigger/GerritManagement/index.jelly

```diff
@@ -1,3 +1,4 @@
+<?jelly escape-by-default='true'?>
```

Comment:
Yes, I took a quick scan through the files that I was changing, the only worry I have is how the data binding works for form values, e.g. if a textfield value contains `"<"` or `">"`?